scout/server/blueprints/variants/templates/variants/components.html

```diff
@@ -96,15 +96,24 @@
           </span>
         {% endif %}
       </a>
-      {% if variant.panels|count > 0 %}
```

Comment:
The code here repeats a lot between SNV and SV, but I left it outside the scope of the PR to merge it.